text-utilities and Lists/typography/typography.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="Laxmikanth Konthum" />
+    <meta name="description" content="Elite UI is a library to provide ready-made components to the user" />
+    <title>Typography</title>
+    <link href="typography.css" rel="stylesheet">
+    <link href="/style.css" rel="stylesheet">
+</head>
+
+<body>
+    <h1 class="text-size-xl"> Headine 1 </h1>
+    <h2 class="text-size-lg"> Headine 2 </h2>
+    <h3 class="text-size-md"> Headine 3 </h3>
+    <h4 class="text-size-sm"> Headine 4 </h4>
+    <h5 class="text-size-xs"> Headine 5 </h5>
+    <p class="text-size-xl"> Text 1 </p>
+    <p class="text-size-lg"> Text 2 </p>
+    <p class="text-size-md"> Text 3 </p>
+    <p class="text-size-sm"> Text 4 </p>
+    <p class="text-size-xs"> Text 5 </p>
```

Comment:
I think no need to write `text-size-xl`. Even if you write `text-xl` it is self-explanatory that we are talking about size Just a small suggestion.

Reply:
Right! makes sense. Will update the code - Thanks Sanket